.github/workflows/ci.yml

```diff
@@ -38,6 +38,9 @@ jobs:
       - name: Check out pipeline code
         uses: actions/checkout@v3
 
+      - name: Setup apptainer
```

Comment:
This step is already in this CI, duplicated.

Reply:
Check TreeVal to see how to run pipeline if you use nf-core download.

https://github.com/sanger-tol/treeval/blob/dd32db5be33201873b765306fd6f8f3eea604494/.github/workflows/ci.yml#L75